src/Testing/Doubles/FakeMollieApiClient.php

```diff
@@ -6,14 +6,21 @@
 
 use Exception;
 use Illuminate\Contracts\Foundation\Application;
+use Mollie\Api\Exceptions\ApiException;
 use Mollie\Api\MollieApiClient;
 use stdClass;
 
 final class FakeMollieApiClient extends MollieApiClient
 {
+    /**
+     * @throws ApiException
+     */
     public static function fake(Application $app): self
     {
-        return $app->instance(self::class, new self());
+        $mollie = new self();
+        $mollie->setApiKey($app['config']->get('mollie.key'));
```

Comment:
Does the Mollie API client now fail if we don't set the ApiKey? Otherwise, I would avoid to do this, or try to use a fake key in order not to accidentally perform calls.